python/sparknlp/internal/extended_java_wrapper.py

```diff
@@ -33,6 +34,8 @@ def apply(self):
         return self._java_obj
 
     def new_java_obj(self, java_class, *args):
+        # check_obj_exist(java_class) TODO is checking this here efficient?
+        # if this is just used for primitive data types, we might want to ommit check here?
```

Comment:
It's not only for primitive types. If we include `check_obj_exist` validation, it will do for all classes defined [here](https://github.com/JohnSnowLabs/spark-nlp/blob/master/python/sparknlp/internal/__init__.py).
In addition, it's used by python classes that read `CoNLL` and `POS` formats (conll.py, conllu.py, pos.py and pub_tator.py)
Not sure about the latency impact. We will need to do performance tests.

Reply:
Thats why I suggest to ommit it  there but keep it for the other places where we just do it once for Anno Classes/  High Level Classes which are usually just created once